### ISSUE TITLE
Increase the value of proof-of-work cryptocurrencies with a tax on mining

### ISSUE THREAD

**Issue by dmarti** [responsible-financial-innovation-act22/RFIA-bill#17]
Proof-of-work (PoW) cryptocurrencies require the use of computing hardware and energy, which are obtained at some cost. The cost of mining limits the amount of the PoW cryptocurrency that can be created, and the PoW cryptocurrency will be sold by miners at a price that reflects the cost to mine it.

A tax on the of costs of mining would increase the market value of the resulting PoW cryptocurrency by at least the amount of the tax -- because the miner would be providing proof of additional "work", the cost of paying the tax.

Because the tax increases the value of the cryptocurrency mined, the miner can recover the amount of taxes paid when they sell the cryptocurrency. So the miner effectively does not pay the tax. And the buyers and users of the PoW cryptocurrency obtain an asset of higher value, because it contains more proof of work, so the tax does not fall on them either.

Taxation of PoW cryptocurrency mining is a rare example of an opportunity to obtain tax revenue at no cost to the parties paying the tax.

**Comment by AtmaTheWanderer** [comment]
The tax in this case is paid by the person purchasing the coins. 

I pay more for something than I would have without the tax existing.  I receive no value benefit for doing so outside of a higher entry point.  I just paid the tax, even if it is collected by and submitted by the mining company.

**Comment by joelself** [comment]
People will just mine in a different country. Sounds like win to me!

**Comment by dmarti** [comment]
The mining company paid the tax, but because it cost them more to mine the coins, the coins are worth more.

Another way to think about it is: what if the taxes on the mining company's electric utility went up? The PoW would represent a greater investment of resources and therefore the PoW coins mined would be worth more. Taxing the mining company has the same effect.

**Comment by bdelgado1995** [comment]
There's a lot of logic in here that doesn't make sense to me. Currently, there are already two taxable events when mining. 1. When you mine the crypto currency. 2. When you sell, which is a gain or loss based on the price of sale and price at the time of mining. Why would a third tax be necessary? 

"the PoW cryptocurrency will be sold by miners at a price that reflects the cost to mine it." Not necessarily, some miners hold indefinitely. 

"A tax on the of costs of mining would increase the market value of the resulting PoW cryptocurrency by at least the amount of the tax -- because the miner would be providing proof of additional "work", the cost of paying the tax." Taxes are paid to the government, how does this increase the crypto market value when that is set by number of buyers/sellers at any given point in time?

**Comment by dmarti** [comment]
I know this is counterintuitive. Here's another way to think about it. Imagine that each miner acquired a free energy machine. The value of each unit of PoW cryptocurrency produced would go down because it represents a lower investment of costly "work".

If the price of electricity went up for a non-tax reason, the value of each unit of PoW cryptocurrency produced would also go up, because it represents proof of more "work."

Proof of work is more accurately described as "proof of expenditure". Most of our classical economics beliefs about the properties of taxable goods do not apply to PoW cryptocurrencies. (The tax is likely to increase the value of the PoW cryptocurrency by _more_ than the cost of the tax on average, if the average miner operates at a profit.)

**Comment by dmarti** [comment]
This is an under-researched but promising area.

 * The tax is not borne by the miner, because they can pass it along to the buyer. (Or, if they hold the cryptocurrency, they hold a more valuable asset.)

 * The tax is not borne by the buyer, because they receive a more valuable unit of cryptocurrency. 

A mining tax on PoW cryptocurrencies could turn out to be the first tax without an ultimate "taxpayer."

**Background:**  https://en.wikipedia.org/wiki/Tax_incidence

**Comment by BillyWM** [comment]
> The cost of mining limits the amount of the PoW cryptocurrency that can be created

Incorrect. The "amount of cryptocurrency that can be created" is algorithmically determined through difficulty adjustments and reward halvings. Difficult adjustments keep the granting rate relatively stable.

The cost of mining limits how much *you*, personally, can create

**Comment by dmarti** [comment]
@BillyWM I will edit this to make it clear that I am referring to the total market value of the crytocurrency, not the number of coins.

**Comment by invino4** [comment]
Wouldn't miners need to pay the tax even if they have not yet realized their profits? They would have to finance the tax either by selling (a portion of) their coin immediately to cover the tax or through debt creating a new solvency risk. At the same time, the increase in coin price would increase the barrier-to-entry for some buyers potentially turning away some buyers and hence reducing demand for the coin overall. The increase in supply from force sales by miners (to cover the tax) and decreased demand (due to higher prices) from buyers would both put downward pressure on the coin's market value. Does that mean that all coin holders would incur the tax burden through smaller rates of return on their investments (i.e., the coin is worth less overall, even with the tax price adjustment)?

**Comment by dmarti** [comment]
@invino4 And yes, the miner who chooses to keep coins can account for them at their current market value, which takes into account the PoW of all expenditures including tax, so would be higher with the tax.
